src/components/Common/Pagination.tsx

```diff
@@ -68,7 +68,7 @@ const Pagination = ({
   };
 
   const totalCount = data.totalCount;
-  if (!totalCount) {
+  if (!totalCount || totalCount < defaultPerPage) {
```

Comment:
shouldn't this be the right solution? correct me if I am wrong

```suggestion
  if (!totalCount || totalCount < rowsPerPage) {
```

Reply:
Yes, apologies